src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public class Rectangle extends Quadrangle {
+    public Rectangle(String color, int width, int height) {
+        super(color, width, height);
+    }
+
+    @Override
+    public double area() {
+        return super.area();
+    }
+
+    @Override
+    public void draw() {
```

Comment:
The same applies to the `draw` method. If the implementation in the `Quadrangle` class does not need to be different for `Rectangle`, you do not need to override it. Otherwise, make sure that the `draw` method here adds specific behavior for the `Rectangle` class.